[DM-5963] Limited number of operations and events loaded (#4614)
SoftwareAG/c8y-docs: content/change-logs/device-management/device-parameter-2-1-0-limited-number-of-operations-and-events-displayed.md

---
date: ""

> title: Limited number of operations and events displayed

Reviewer: ```suggestion
title: Improved page navigation of device parameter details
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--LJtTuzaN
    label: Device Parameter
build_artifact:
  - value: tc-wfTX6sxsr
    label: device-parameter
ticket: DM-5963
version: 2.1.0
---

> An update has been made to the Device Parameter details view so that the automatic loading of historical operations and events is replaced by a manual **Load More** button. Previously, the seamless fetching of older entries often prevented the user from scrolling down to subsequent parameters in the list, as the page length was continuously extended. With this change, older data is only retrieved upon an explicit click, ensuring that page navigation remains consistent and that subsequent parameters can be reached without interruption.

Reviewer: ```suggestion
In the device parameter details, the automatic loading of historical operations and events has been replaced by a **Load More** button. Previously, the seamless fetching of older entries often prevented the user from scrolling to subsequent parameters in the list, as the page length kept increasing. With this change, older data is only retrieved upon an explicit click, ensuring that page navigation remains consistent and that subsequent parameters can be reached without interruption.
```
